builtins/web/blob.h

```diff
@@ -51,13 +51,19 @@ class Blob : public TraceableBuiltinImpl<Blob> {
   static const JSPropertySpec properties[];
 
   static constexpr unsigned ctor_length = 0;
-  enum Slots { Data, Type, Endings, Readers, Count };
+  enum Slots { Data, Type, Endings, Readers, Reserved1, Reserved2, Count };
```

Comment:
Since these are being reserved _by_ the FIle API, it may just make sense to directly call these `MaybeFileName` and `MaybeFileModified`, since those are their only meanings. Alternatively explicitly note they are reserved by the File API - `FileAPISlot1` and `FileAPISlot2`.

Reply:
Also I'm curious why not just have a `File` slot and have it point to a file instance here, where the File builtin has the `Name` and `Modified` slots?